src/components/CalendarEventsDialog.vue

```diff
@@ -407,13 +412,13 @@ async function submitNewMeeting() {
 						{{ option.label }}
 					</template>
 				</NcSelect>
-				<h5 class="calendar-meeting__header">
+				<h5 v-if="!isOneToOneConversation" class="calendar-meeting__header">
 					{{ t('spreed', 'Attendees') }}
 				</h5>
 				<NcCheckboxRadioSwitch v-model="selectAll" @update:modelValue="toggleAll">
-					{{ t('spreed', 'Invite all users and emails') }}
+					{{ !isOneToOneConversation? t('spreed', 'Invite all users and emails') : t('spreed', 'Invite user') }}
```

Comment:
We just recently discussed with design team, that 'user' should be avoided. How about:
```suggestion
					{{ !isOneToOneConversation ? t('spreed', 'Invite all users and emails') : t('spreed', 'Invite {user}', user: <secondParticipant.displayName>) }}
```
And move it to script?